app/frontend/actions/delegate.ts

```diff
@@ -131,10 +144,71 @@ export default (store: Store) => {
     })
   }
 
+  const delegate = async (state: State): Promise<void> => {
+    return await confirmTransaction(state, {
+      txConfirmType: TxType.DELEGATE,
+      txPlan: state.cachedTransactionSummaries[TxType.DELEGATE].plan,
+      sourceAccountIndex: state.sourceAccountIndex,
+    })
+  }
+
+  const deregisterStakingKey = async (state: State): Promise<void> => {
+    const supportError = getWallet().ensureFeatureIsSupported(CryptoProviderFeature.WITHDRAWAL)
```

Comment:
ensureFeatureIsSupported seems to not return anything as per the typing: https://github.com/vacuumlabs/adalite/blob/e4cde52738dbb2e224f1d6df68a13a3113c12446/app/frontend/types.ts#L44

Reply:
ah I see, I should be looking at the function with the same name, but declared in shelley-wallet.ts :facepalm: - this is madness :D If it returns a boolean, something like isFeatureSupported would imho be more spot-on as a name